imports/plugins/included/default-theme/client/styles/badge.less

```diff
@@ -63,7 +63,6 @@
   border: 1px solid @brand-warning;
 }
 
-
```

Comment:
Let's not add/blank lines for no reason

Reply:
Ok. I just pushed a commit that fixes that.

Reply:
Still wasn't changed